assets/locales/service.en.toml

```
@@ -47,3 +47,19 @@ one = "Released after"
 [DateFilterDescription]
 description = "For example: 2006 or 19/07/2010"
 one = "For example: 2006 or 19/07/2010"
+
+[ReleaseCalendarFiltersTitle]
+description = "Filter results"
+one = "Filter results"
```

Wouldn't the singular for this be `Filter result`
```suggestion
one = "Filter result"
other = "Filter results"
```

---

Yes it would :)